common/config.py

```diff
@@ -18,3 +19,54 @@
 # Constants used to define benchmarking standards.
 NUM_ROWS = [10]  # The column lengths to use for benchmarked objects
 NUM_COLS = [1, 6]  # The numbers of columns to use for benchmarked DataFrames
+
+
+def flatten(xs):
+    for x in xs:
+        if not isinstance(x, str):
+            yield from x
+        else:
+            yield x
+
+
+def cudf_benchmark(cls, dtype="int", nulls=None, cols=None, name="obj"):
```

Comment:
Let's use keyword-only arguments here for `dtype`, `nulls`, etc.?

Reply:
Done.